assets/src/scripts/store.js

```diff
@@ -103,13 +127,26 @@ export const Actions = {
             parameterCode,
             variableID
         };
+    },
+    setGageHeight(gageHeightIndex) {
+        return {
+            type: 'SET_GAGE_HEIGHT',
+            gageHeightIndex
+        };
     }
 };
 
 
 export const timeSeriesReducer = function (state={}, action) {
     let newState;
     switch (action.type) {
+        case 'SET_FLOOD_FEATURES':
+            return {
+                ...state,
+                floodStages: action.stages,
+                floodExtent: action.extent,
+                gageHeight: action.stages.length > 0 ? action.stages[0] : null
+            };
```

Comment:
It would be nice to have a root reducer with separate reducers for each component - would that be hard to do as part of this story?

Reply:
I think we should open a new ticket for that. I've spent a lot of time on this ticket already